dataset_loader/src/main.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+/*
+* Copyright (C) 2022 Intel Corporation
+* SPDX-License-Identifier: Apache-2.0
+*/
+
+#include "hebench_dataset_loader_functional_test.h"
+
+int main(int argc, char **argv)
+{
+    if (argc < min_args_allowed)
```

Comment:
This works, but it may be cleaner to use the args parser provided in common lib.

Reply:
Ernesto,

Thanks for the input and I think you're right, but also think it does not hurt to leave it that way since it will only be used by the CI and also won't be using any other piece of SW but the dataset_loader.

Reply:
OK. Sounds good. If this software or any other becomes part of the framework, let's make sure they re-use existing code.